dse-next/README.md

```diff
@@ -0,0 +1,28 @@
+## dse-next Docker image
```

Comment:
Thanks for adding this.

Reply:
No problem. Realized that after writing down for myself no reason it couldn't be shared.